pkg/cluster/cluster.go

```diff
@@ -52,6 +52,15 @@ func NewFromFile(path string) (*Cluster, error) {
 	return NewFromYAML(data)
 }
 
+// ConfigExists checks whether a configuration file has already been created.
+// Returns false if not true if it already exists.
+func (c *Cluster) ConfigExists(path string) bool {
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return false
+	}
+	return true
```

Comment:
I don't think this function should be on the cluster object, it's not specific to a cluster, it's just checking if a file exists and should probably be a function in create.go.

There's an issue here: if `stat` fails, but not because the file doesn't exist (for instance it could fail because we can't access the path we're trying to stat), we'll happily return that the file exists and do nothing. In that case, we'd want to answer `false` and let the code creating the file fail, giving the proper error (eg. permission denied). So something like the below would do the trick /me thinks:

```
	_, err := os.Stat(path); {
		return false
	}
       return err != nil
```

Reply:
In Go, it's been a long outstanding debate on how to check whether a file exists or not. :) in the end, you want to check whether it's a nonexists error. This is also how Go does it internally.

What other you can do is if it returns exists then check if it's a directory. os.Stat can fail of different reasons, you don't want to just return whatever it failed with.

I'll adjust it.

Also, I was debating where to put this. 🤔  I ended up with cluster, because that's also where it saves the config. so I thought it's related. :)

Reply:
I added the permission check although they usually don't do this. I have to dig up some ancient forum records to find out why. I swear I remember it being part of something that's the reason you usually don't find it anywhere. :)

Reply:
Okay, so apparently, permissions aren't an issue for a stat. Basically, if there are permission problems, they would only surface upon writing. Stat it self will not fail even with the now added IsPermission check. 🤔  I'll leave it if it makes you feel better. :D